src/test/java/com/cloudbees/plugins/credentials/CredentialsProviderTest.java

```diff
@@ -278,7 +276,7 @@ public void testNodeCredentialFingerprintsAreRemovedForNonExistentNodes() throws
         // it should not be recorded
         DumbSlave nonAddedSlave = new DumbSlave("non-added-slave",
                 "dummy", "/home/test/agent", "1", Node.Mode.NORMAL, "remote",
-                new JNLPLauncher(),
+                new JNLPLauncher(false),
```

Comment:
This was a bad change in core BTW. The no-arg ctor should have been retained, and `enableWorkDir` made into a `@DataBoundSetter`, or just removed altogether since it is a derivative concept. Way on the back burner to fix, but I would suggest reverting this and just suppressing the deprecation warning.

Reply:
Both done.